src/libraries/System.Text.Json/tests/System.Text.Json.Tests/TrimmingTests/IsReflectionEnabledByDefaultFalse.cs

```diff
@@ -44,18 +44,41 @@ public static int Main()
         {
         }
 
+        // Calling GetConverter should throw NotSupportedException.
+        try
+        {
+            _ = options.GetConverter(typeof(MyPoco));
```

Comment:
@eerhardt @vitek-karas for context to https://github.com/dotnet/runtime/pull/90013/files#r1284314974 adding this method resulted in rooting the reflection resolver.

Reply:
Would you have a branch/test we can look at?
I agree that the feature switch should make this work, so the fact that it doesn't is interesting.
(Adding @sbomer as I'm going to be OOF next week)

Reply:
Sure, here's one: https://github.com/dotnet/runtime/commit/b2ba402683d34c58cd486814904b6391ed954703

Reply:
I just chanced on this discussion again, has this been looked at? I could create an issue if necessary.